src/InitSettings.jsx

```diff
@@ -24,55 +23,37 @@ const Item = styled(Paper)(({ theme }) => ({
 
 function InitSettings() {
   const {
-    walking, setKinematicValue, setWalkingValue,
+    GRPC_WEB_API_URL, published, setPublished, setKinematic, setWalking,
   } = useContext(WalkContext);
 
-  const client = useClient();
-  const logger = useLogger();
+  const client = new aruku_interfaces.ConfigClient(GRPC_WEB_API_URL, null, null);
+  const request = new aruku_interfaces.Empty();
 
-  const [fetching, handleFetch] = useHandleProcess(() => client
-    .call({})
-    .then((response) => {
-      logger.success('Successfully get config.');
-      const kinematicData = JSON.parse(`${response.json_kinematic.replace('/\\/g', '')}`);
-      const walkingData = JSON.parse(`${response.json_walking.replace('/\\/g', '')}`);
-      Object.keys(kinematicData).map((name) => Object.keys(kinematicData[name])
-        .map((key) => setKinematicValue(name, key, kinematicData[name][key])));
-      Object.keys(walkingData).map((name) => Object.keys(walkingData[name])
-        .map((key) => setWalkingValue(name, key, walkingData[name][key])));
-    })
-    .catch((err) => {
-      logger.error(`Failed to load config! ${err.message}.`);
-    }), 500);
+  const handleFetch = () => {
+    client.getConfig(request, {}, (err, response) => {
+      if (err) {
+        console.log(`Unexpected error: code = ${err.code}` + `, message = "${err.message}"`);
```

Comment:
```suggestion
        console.log(`Unexpected error: code = ${err.code}, message = "${err.message}"`);
```

Reply:
yea I found this problem when copying components to Ichiro-app, I think you should fix all these same problems @FaaizHaikal